sw/airborne/firmwares/rotorcraft/guidance/guidance_h.c

```diff
@@ -555,6 +555,7 @@ void guidance_h_nav_enter(void)
   reset_guidance_reference_from_current_position();
 
   nav_heading = stateGetNedToBodyEulers_i()->psi;
+  guidance_h.sp.heading = stateGetNedToBodyEulers_f()->psi;
```

Comment:
Ah yes, so for this commit this is not essential. The reason for this is that I would like to have the option to not use nav_heading, as there are vehicles for which this is not a free variable (hybrids for instance). Then guidance_h.sp.heading needs to be reset on enter, as it will not be set from the (reset) nav_heading.
I thought for now this would not hurt.